Setup.sh

```diff
@@ -131,211 +149,167 @@ PIPELINE_CI=1
-		if [ "$variable" ]; then
-			declare DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORD="${variable}"
-		fi
-		read -p "Specify the DBB Db2 Metadatastore JDBC Password File [leave empty if not used]: " variable
-		if [ "$variable" ]; then
-			declare DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORDFILE="${variable}"
-		fi
-		if [ "$DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORD" = "" ] & [ "$DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORDFILE" = "" ]; then
-			echo "[ERROR] Either the Db2 JDBC User Password or the Db2 JDBC Password File must be specified. Exiting."
-			rm -rf $DBB_MODELER_WORK
-			exit 1
-		fi
+	read -p "Specify the DBB Db2 Metadatastore JDBC Password File [default: ${DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORDFILE} ]: " variable
+	if [ "$variable" ]; then
+		declare DBB_MODELER_DB2_METADATASTORE_JDBC_PASSWORDFILE="${variable}"
 	fi
 fi
 
-if [ $rc -eq 0 ]; then
-	# Specify input files
-	echo
-	echo "[SETUP] Specifying DBB Git Migration Modeler input configuration"
-	for config in ${input_array[@]}; do
-		read -p "Specify input parameter $config [default: ${!config}]: " variable
-		if [ "$variable" ]; then
-			declare ${config}="${variable}"
-		fi
-	done
-	echo "Specify the pipeline orchestration technology to use."
-	read -p "1 for 'AzureDevOps', 2 for 'GitlabCI', 3 for 'Jenkins' or 4 for 'GitHubActions' [default: 1]: " variable
+# Specify input files
+echo
+echo "[SETUP] Specifying DBB Git Migration Modeler input configuration"
+for config in ${input_array[@]}; do
+	read -p "Specify input parameter $config [default: ${!config}]: " variable
 	if [ "$variable" ]; then
-		declare PIPELINE_CI="${variable}"
-	else
-		declare PIPELINE_CI="1"
+		declare ${config}="${variable}"
 	fi
-	case ${PIPELINE_CI} in
-	"1")
-		PIPELINE_CI="AzureDevOps"
-		;;
-	"2")
-		PIPELINE_CI="GitlabCI"
-		;;
-	"3")
-		PIPELINE_CI="Jenkins"
-		;;
-	"4")
-		PIPELINE_CI="GitHubActions"
-		;;
-	esac
+done
+echo "Specify the pipeline orchestration technology to use."
+read -p "1 for 'AzureDevOps', 2 for 'GitlabCI', 3 for 'Jenkins' or 4 for 'GitHubActions' [default: 1]: " variable
+if [ "$variable" ]; then
+	declare PIPELINE_CI="${variable}"
+else
+	declare PIPELINE_CI="1"
+fi
+case ${PIPELINE_CI} in
+"1")
+	PIPELINE_CI="AzureDevOps"
+	;;
+"2")
+	PIPELINE_CI="GitlabCI"
+	;;
+"3")
+	PIPELINE_CI="Jenkins"
+	;;
+"4")
+	PIPELINE_CI="GitHubActions"
+	;;
+esac
 
-	echo
-	echo "[SETUP] Copying DBB Git Migration Modeler sample configuration files to '$DBB_MODELER_WORK'"
-	if [ ! -d "${DBB_MODELER_APPMAPPINGS_DIR}" ]; then
-		mkdir -p $DBB_MODELER_APPMAPPINGS_DIR
-		rc=$?
-	fi
-	if [ $rc -eq 0 ]; then
-		cp $DBB_MODELER_HOME/samples/applications-mapping/*.* $DBB_MODELER_APPMAPPINGS_DIR/
-		rc=$?
-	fi
-	if [ $rc -eq 0 ]; then
-		cp $DBB_MODELER_HOME/samples/repositoryPathsMapping.yaml $REPOSITORY_PATH_MAPPING_FILE
-		rc=$?
+echo
+DBB_GIT_MIGRATION_MODELER_CONFIG_FILE="DBB_GIT_MIGRATION_MODELER-$(date +%Y-%m-%d.%H%M%S).config."
```

Comment:
Removed the `dot` at the end. 

```suggestion
DBB_GIT_MIGRATION_MODELER_CONFIG_FILE="DBB_GIT_MIGRATION_MODELER-$(date +%Y-%m-%d.%H%M%S).config"
```